terraform/digitalocean/k8s/outputs.tf

```diff
@@ -5,3 +5,7 @@ output "cluster_status" {
 output "cluster_endpoint" {
   value = digitalocean_kubernetes_cluster.k8s.endpoint
 }
+
+output "kubernetes_config" {
+  value = substr(var.config_output_path, -1, 1) == "/" ? "${var.config_output_path}kubeconfig_${var.cluster_name}" : var.config_output_path
```

Comment:
[p2] Please, describe logic of this high complexity line as comment

Reply:
Outpyt aligned with Terraform EKS module:
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/312596f46d937c74143ce9a9563438b2b52f0f0e/kubectl.tf
Added comment
 # If first symbol defines absolute path (/) - form fullname to config, if not - use provided